src/App.css

```diff
@@ -1,38 +1,183 @@
-.App {
-  text-align: center;
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500&display=swap');
+
+  * {
```

Comment:
@abinsinguza-dallen , please ensure to pull the upstream branch before pushing so as to avoid merge conficts